pkg/v1/tkg/test/tkgctl/docker_cc/docker_suite_test.go

```diff
@@ -0,0 +1,214 @@
+	Expect(e2eConfigPath).ToNot(BeNil(), "Failed to load e2e config from %s", e2eConfigPath)
+
+	logLocation := filepath.Join(artifactsFolder, "logs")
+	hackCmd := exec.NewCommand(
+		exec.WithCommand("../../scripts/cc_hack.sh"),
+		exec.WithStdout(GinkgoWriter),
+	)
+
+	fmt.Println("Executing the hack script")
+	out, cmdErr, err := hackCmd.Run(context.Background())
+	fmt.Println(string(out))
+	fmt.Println(string(cmdErr))
+	Expect(err).To(BeNil())
+
+	cli, err := tkgctl.New(tkgctl.Options{
+		ConfigDir: e2eConfig.TkgConfigDir,
+		LogOptions: tkgctl.LoggingOptions{
+			File:      filepath.Join(logLocation, "before_suite.log"),
+			Verbosity: e2eConfig.TkgCliLogLevel,
+		},
+	})
+	Expect(err).To(BeNil())
+
+	// save config variables from e2e config to the tkg config file
+	err = e2eConfig.SaveTkgConfigVariables()
+	Expect(err).To(BeNil())
+
+	timeout, err := time.ParseDuration(e2eConfig.DefaultTimeout)
+	Expect(err).To(BeNil())
+
+	// create management cluster
+	if !e2eConfig.UseExistingCluster {
+		err := cli.Init(tkgctl.InitRegionOptions{
+			Plan:                        e2eConfig.ManagementClusterOptions.Plan,
+			ClusterName:                 e2eConfig.ManagementClusterName,
+			InfrastructureProvider:      e2eConfig.InfrastructureName,
+			Timeout:                     timeout,
+			Size:                        e2eConfig.ManagementClusterOptions.Size,
+			DeployTKGonVsphere7:         e2eConfig.ManagementClusterOptions.DeployTKGonVsphere7,
+			EnableTKGSOnVsphere7:        e2eConfig.ManagementClusterOptions.EnableTKGSOnVsphere7,
+			VsphereControlPlaneEndpoint: e2eConfig.ManagementClusterOptions.Endpoint,
+			CniType:                     "antrea",
+			Edition:                     "tkg",
+		})
+
+		Expect(err).To(BeNil())
+	}
+
+	//// Create initial workload cluster
+	//clusterName = e2eConfig.ClusterPrefix + "wc"
+	//options := framework.CreateClusterOptions{
+	//	ClusterName: clusterName,
+	//	Namespace:   constants.DefaultNamespace,
+	//	Plan:        "dev",
+	//}
+	//clusterConfigFile, err := framework.GetTempClusterConfigFile(e2eConfig.TkgClusterConfigPath, &options)
+	//Expect(err).To(BeNil())
+	//
+	//defer os.Remove(clusterConfigFile)
+	//
+	//tkgCtlClient, err := tkgctl.New(tkgctl.Options{
+	//	ConfigDir: e2eConfig.TkgConfigDir,
+	//	LogOptions: tkgctl.LoggingOptions{
+	//		File:      filepath.Join(logsDir, clusterName+".log"),
+	//		Verbosity: e2eConfig.TkgCliLogLevel,
+	//	},
+	//})
+	//Expect(err).ToNot(HaveOccurred())
+	//
+	//optOutStatus, err := tkgCtlClient.GetCEIP()
+	//Expect(err).ToNot(HaveOccurred())
+	//Expect(optOutStatus.CeipStatus).To(Equal("Opt-out"))
+	//
+	//By(fmt.Sprintf("Creating initial workload cluster %q", clusterName))
+	//
+	//defer os.Remove(clusterConfigFile)
+	//err = tkgCtlClient.CreateCluster(tkgctl.CreateClusterOptions{
+	//	ClusterConfigFile: clusterConfigFile,
+	//	Edition:           "tkg",
+	//})
+	//Expect(err).To(BeNil())
```

Comment:
Is there a reason to keep this as commented or should we remove this?

Reply:
Deleted